kompendium-core/src/main/kotlin/io/bkbn/kompendium/models/oas/OpenApiSpecComponentSchema.kt

```diff
@@ -1,6 +1,13 @@
 package io.bkbn.kompendium.models.oas
 
-sealed class OpenApiSpecComponentSchema(open val default: Any? = null) {
+import kotlinx.serialization.Contextual
+import kotlinx.serialization.Serializable
+
+@Serializable
+sealed class OpenApiSpecComponentSchema(
+  @Contextual
+  open val default: Any? = null
```

Comment:
Ahh unfortunately this line makes the kotlinx serialization approach not feasible... as it would require any default value provided to have a corresponding kotlinx serializer

Reply:
we may have to go with Jackson as the underlying json writer because it is the most flexible of the bunch